pkg/webhook.go

```diff
@@ -37,10 +39,13 @@ var _ admission.CustomValidator = &Webhook{}
 func (w *Webhook) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
 	allow, err := w.state.ReadConfig(ctx)
 	if err != nil {
+		w.metrics.Denied.Inc()
```

Comment:
Very minor: for readability I'd suggest moving this logic in a separate function and end-up with something like

```golang
if err := myfunc(); err != nil {
  deny++
  return nil, err
}
accepted++
return nil, nil
```

- sorry for the bad code I'm from mobile 😁

Reply:
I don't understand how this is any more readable than stuffing it all into the `ValidateCreate` method, since we don't care about validating updates or deletions.  Could you explain your reasoning here?

Reply:
Outdated with 659b5be.